### PR TITLE
Pull request for desktop version, add csss ad html code with styles and sizing 

### PR DESCRIPTION
**Pull request by TechnoAndy**
Pull request for desktop version of portfolio. Added media query to css code.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,104 +11,49 @@
...
-                <button class="btn" type="button">See Project</button>
-              </div>
-            </div>
-              <div class="work-card">
-                <div>
-                  <img class="work-card-img" src="SnapshootPortfoliotonic01.png" alt="" />
+              <div class="container">
+                <div class="work-card">
+                  <div class="img">
+                  <img src="images/SnapshootPortfolio02.png" alt="" />
```
Comment (mahbubul14) [inline index.html]: - [x] Kindly change the background color of this section to match the Figma design. :+1: 

- [x] Please adjust the margin and padding of the image to match the Figma design. :+1: 

- [x] Kindly reduce the gap between the content and image of this section. :+1: 

- [x] You should change the font color and weight to match with the Figma design. (ex: project descriptions, technologies, see project button). :+1: 

- [x] The "See Project" button should take you to the Live Demo of the project or to the GitHub repository. Kindly work on it. :+1: 

Your design            
:-------------------------:
Screenshot from 2022-08-27 02-25-24


|  Figma Design
|:-------------------------:

Screenshot from 2022-08-27 02-28-09

File: index.html [hunk 2]
```diff
@@ -122,80 +67,144 @@ <h3 class="work-heading-2">Multi-Post Stories</h3>
...
+          </ul>
+          <button class="btn" type="button">See Project</button>
+        </div>
+      </div>
             </section>
               <section class="about-section-2">
                 <div class="align-center-text-2">
-                  <h2 class="header-text">About<br>Myself</h2>
+                  <div class="about-detail-left">
+                    <h2 class="header-text">About<br>Myself</h2>
```
Comment (mahbubul14) [inline index.html:135]: - [x] You should reduce the gap between the left and right blocks of this section. :+1: 

- [x] Your current design for the right block doesn't match with the Figma design. Kindly increase the gap between the dropdown menu and experiences. :+1: 

Your design  
:-------------------------:
Screenshot from 2022-08-27 02-41-21

Figma Design
|:-------------------------:
Screenshot from 2022-08-27 02-45-21

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
Hi @TechnoAndy ,

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

# Required Changes ♻️

- [ ] Your current readme file is not professional enough. Kindly add the getting started section. This should be an example of how you may give instructions on setting up your project locally and also update the author section properly. You can use [this template ](https://github.com/microverseinc/readme-template)for the `README` but remember to customize it to your project.

- [ ] Kindly check the inline comments under the review. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linter error.
:heavy_check_mark: Github flow used properly.
:heavy_check_mark: Descriptive PR title.

### Optional suggestions

- [ ] Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by tugsem — APPROVED** [review]
Hi  @TechnoAndy , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Alt Text

## Highlights ⭐ 
✅ You have applied requested changes
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**